torch/csrc/jit/mobile/upgrader_mobile.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+// #include <ATen/core/ivalue.h>
+#include <ATen/core/ivalue_inl.h>
+
+#include <string>
+#include <vector>
+
+#include <torch/csrc/jit/serialization/import_export_functions.h>
+#include <unordered_map>
+
+namespace torch {
+namespace jit {
+
+struct Upgrader {
+  int min_version;
+  int max_version;
+  std::string upgrader_name;
+  int index;
+};
+
+// From operator_versions.yaml
+const std::unordered_map<std::string, std::vector<Upgrader>> kOperatorVersionMap(
+    {{std::string("aten::div_Tensor"),
+      std::vector<Upgrader>({Upgrader({0, 3, "div_Tensor_0_3", 0})})}});
+
+static const std::vector<std::pair<std::string, c10::IValue>> kUpgraderBytecode(
+    {
+        // The following are called from setup sections.
+        {"div_Tensor_0_3",
+         to_tuple(std::vector<c10::IValue>{
```

Comment:
It's quite space consuming. Could you directly build a mobile::Function here, instead of those strings and later parsed to a function?

Reply:
Address in the latest commit.